### DISCUSSION TITLE
When add a domain name in "Direct/Proxy Domain List", will its sub-domain name follow the same behavior?

### DISCUSSION THREAD

**Discussion by evan0greenup** [immortalwrt/homeproxy#246] [Q&A] [answered]
In `Homeproxy -> Client Setting -> Access Control -> <Proxy Domain List or Direct Domain List>`, When adding a domain name for example `aws.amazon.com` into `Direct Domain List`, will `console.aws.amazon.com` also follow the same strategy?

**Comment by 1715173329** [comment, marked as answer]
Correct.
